scripts/build_food_db.py

```
@@ -71,35 +101,46 @@ def load_source_data(self) -> tuple[List[Dict], List[Dict]]:
         try:
             usda_data = list(adapter.normalize())
             print(f"  ✅ USDA: {len(usda_data)} records")
-        except Exception as e:
-            print(f"  ❌ USDA error: {e}")
+        except (FileNotFoundError, ValueError, OSError) as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)